.github/workflows/package.yaml

```
@@ -53,14 +53,33 @@ jobs:
+
+      - run: python -m unittest -fv
+
+  test:
+    needs: [build]
+    strategy:
+      matrix:
+        os: [macos-latest, ubuntu-latest]
+        python-version: ["3.8", "3.11"]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)